frb_codegen/src/generator/dart/mod.rs

```diff
@@ -514,6 +540,22 @@ fn generate_wire2api_func(
     )
 }
 
+fn generate_opaque_func(ty: &IrType) -> Acc<String> {
+    Acc {
+        io: format!(
+            "late final ffi.NativeFinalizer _{0}Finalizer = ffi.NativeFinalizer(inner._drop_opaque_{0}Ptr);
+            ffi.NativeFinalizer get {0}Finalizer => _{0}Finalizer;",
+            ty.dart_api_type(),
+        ),
+        wasm: format!(
+            "late final Finalizer<int>  _{0}Finalizer = Finalizer(inner.drop_opaque_{0});
```

Comment:
problem: Worries whether `Finalizer<int>` really works? Because this means "when a *int* is GCed, finalizer will be called". But not sure whether a *int* will ever be GCed.

Reply:
`Finalizer<int>`, where `int` - token that is passed to the `Finalizer` function. The finalizer is attached directly to the RustOpaque instance.

Reply:
Get it. Then totally ok